boards/lilygo/ttgo_t8s3/doc/index.rst

```diff
@@ -0,0 +1,238 @@
+.. _ttgo_t8s3:
+
+Lilygo TTGO T8S3
+################
+
+Overview
+********
+
+Lilygo ``ttgo_t8s3`` is an IoT mini development board based on the
+Espressif ESP32-S3 WiFi/Bluetooth dual-mode chip.
+
+It features the following integrated components:
+
+- ESP32-S3 chip (240MHz dual core, Bluetooth LE, Wi-Fi)
+- on board antenna and IPEX connector
+- USB-C connector for power and communication
+- MX 1.25mm 2-pin battery connector
+- JST SH 1.0mm 4-pin UART connector
+
+.. figure:: img/ttgo_t8s3.webp
+   :align: center
+   :alt: TTGO T8S3
+
+   Lilygo ``ttgo_t8s3``
+
+Functional Description
+**********************
+This board is based on the ESP32-S3 with 16MB of flash, WiFi and BLE support. It
+has an USB-C port for programming and debugging, integrated battery charging
+and an on-board antenna. The fitted U.FL external antenna connector can be
+enabled by moving a 0-ohm resistor.
+
+Connections and IOs
+===================
+
+The ``ttgo_t8s3`` board configuration supports the following hardware features:
```

Comment:
board target

Reply:
changed.

Reply:
> board target

@nordicjm 
FWIW until this is fixed for all boards across the tree this will keep coming back as people typically copy-paste boilerplate from existing boards.

https://docs.zephyrproject.org/latest/boards/actinius/icarus_bee/doc/index.html#supported-features
https://docs.zephyrproject.org/latest/boards/fanke/fk7b0m1_vbt6/doc/index.html#supported-features
https://docs.zephyrproject.org/latest/boards/vngiotlab/nrf52_vbluno52/doc/index.html#supported-features
... and many, many, more